src/devices/Tsl256x/Tsl256x.cs

```diff
@@ -0,0 +1,353 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Buffers.Binary;
+using System.Device.I2c;
+using System.Threading;
+using UnitsNet;
+using UnitsNet.Units;
+
+namespace IoT.Device.Tsl256x
+{
+    /// <summary>
+    /// Light to Digital Converter TSL2560 and TSL2561
+    /// </summary>
+    public class Tsl256x : IDisposable
+    {
+        // All information from datasheet https://cdn-shop.adafruit.com/datasheets/TSL2561.pdf
+        private I2cDevice _i2cDevice;
+        private IntegrationTime _integrationTime;
+        private Gain _gain;
+        private InterruptControl _interruptControl;
+        private InterruptPersistence _interruptPersistence;
+        private PackageType _packageType;
+
+        /// <summary>
+        /// When the address select pin is to ground
+        /// </summary>
+        public const int SecondI2cAddress = 0x29;
```

Comment:
I think you should swap the order so that Default is first then second, then third 😄

Reply:
The default is the 0x39, the second and third has no importance :-)

Reply:
I meant the field order :)

Reply:
:-)))) I can do that, yes :-)